Makefile

```diff
@@ -27,38 +33,61 @@ else
 PLATFORM := kubernetes
 endif
 
-all: manager
+# our targets don't generate files in a way make assumes
+.PHONY: all clean test compile prepare run run_as_current_user debug deploy undeploy generate_deployment manifests fmt vet generate docker-build docker-push controller-gen help
+
+all: compile
+
+### clean: deletes the potential leftover generated files from the previous build runs
+clean:
+	@rm -Rf $(TEMP_DIR)
 
 ### test: Run tests
 test: generate fmt vet manifests
```

Comment:
Should we link this to https://github.com/che-incubator/devworkspace-che-operator/blob/main/.github/workflows/test.yml#L64 ?

Reply:
we could

Reply:
make sense to do it with this pr or we can postpone that?

Reply:
I did it in this PR. See the later commits.